config.json

```diff
@@ -6,14 +6,6 @@
   "deprecated": [
 
   ],
-  "ignored": [
-    "_template",
-    "bin",
-    ".gradle",
-    "docs",
-    "img",
-    "build"
```

Comment:
It's generated as part of the run-all-the-unit-tests script, but it's ignored so should never appear in this repo. I'll add a delete step to the unit tests script to avoid configlet getting upset on repeated runs of said script. (I think that only affects local development.)